elements/lisk-chain/src/chain.ts

```diff
@@ -347,7 +347,7 @@ export class Chain {
 			throw invalidTransactionResponse.errors;
 		}
 
-		validatePayload(block, this.constants.maxPayloadLength);
+		validateTransactionRoot(block, this.constants.maxPayloadLength);
```

Comment:
method name and the input doesn't match here

Reply:
`validateBlockProperties`?

Reply:
validatePayload was accurate? i think `BlockProperties` includes other things